src/clients/security_auto.rb

```diff
@@ -90,37 +90,37 @@ def main
         end
         @ret = Security.Import(
           Map.KeysToUpper(
-            Convert.convert(@param, :from => "map", :to => "map <string, any>")
+            Convert.convert(@param, from: "map", to: "map <string, any>")
```

Comment:
in fact whole this convert can be dropeed as implementation do not distinct between map and specialized map.